src/js/user_activity.js

```diff
@@ -0,0 +1,55 @@
+var interval = 0;
+
+function is_user_alive () {
+	if ($("body").hasClass("toc-expanded") || $("body").hasClass("sitenav-expanded")) {
+		interval = 0;
+	}
+	else {
+		interval+= 100;
```

Comment:
Why can't you simply set a timer to 1000 and reset it everytime the user does something ? The logic in there seems more complicated than we need :)